.github/workflows/README.md

```diff
@@ -0,0 +1,69 @@
+# CI Workflows
+This project contains two GitHub Action workflows that are performed following various triggers. For more information on Github Actions see [here](https://docs.github.com/en/actions).
+
+## openactive-test-suite.yml
+
+
+This workflow defines the following jobs: 
+- **test-server**
+- **test-fake-database**
+- **core**
+- **framework**
+- **deploy-reference-implementation**
+- **publish-server**
+- **publish-fake-database**
+
+### test-server job
+This job builds and runs the `OpenActive.Server.NET.Tests` project. 
+
+### test-fake-database job
+This job is almost identical to the `test-server` job, but instead of building and running `OpenActive.Server.NET.Tests`, it builds and runs `Fakes/OpenActive.FakeDatabase.NET.Tests`.
+
+### core job
+This job builds and runs the `OpenActive.Server.NET` project. Then it checks out Test Suite and runs the matrix of tests the `OpenActive.Server.NET` project.
+
+### framework job
+This job is very similar to the above but builds `OpenActive.Server.NET` using .NET Framework and runs Test Suite against it.
+
+
+### deploy-reference-implementation job
+This job deploys the `master` branch of Reference Implementation Booking Server and Identity Server Reference Implementation to two Azure webapps
+
+
+### publish-server job
+This job publishes the OpenActive.Server.NET project to Nuget. It then creates a release in the GitHub repository.
+
+
+### publish-fake-database job
+Very similar to the above `publish-server` job, this job publishes the Fake Database project within OpenActive.Server.NET to Nuget.
+
+## create-dependencies-pr.yaml
+
+This workflow updates OpenActive.Server.NET's OA dependencies if there has been a change in one of the projects. This therefore keeps Server.NET up-to-date with the rest of the OA ecosystem.
+
+This workflow can be triggered in two ways:
+
+- **workflow_dispatch**: This allows you to manually trigger the workflow through the GitHub Actions UI.
+
+- **repository_dispatch**: This allows the workflow to be triggered when a custom repository dispatch event is sent. It listens for dispatch events from two OA repos, specifically when they are updated: `OpenActive.NET-update` and `OpenActive.DatasetSite.NET-update`. 
```

Comment:
This is probably just because of my lack of knowledge, but I don't understand this